src/Components/ServiceScene/Breakdowns/LabelBreakdownScene.tsx

```diff
@@ -273,34 +272,56 @@ export class LabelBreakdownScene extends SceneObjectBase<LabelBreakdownSceneStat
     navigateToValueBreakdown(ValueSlugs.label, value, serviceScene);
   };
 
-  public static Component = ({ model }: SceneComponentProps<LabelBreakdownScene>) => {
-    const { body, loading, blockingMessage, error, search, sort } = model.useState();
+  public static ParentMenu = ({ model }: SceneComponentProps<LabelBreakdownScene>) => {
```

Comment:
For clarity, I would consider calling these `BreakdownMenu` and `ValuesMenu` or even `LabelsMenu` and `ValuesMenu`.

Reply:
Fixed: see https://github.com/grafana/explore-logs/pull/936/commits/8feec08b0b26cb0773e299722172e14fe1ace406